pkg/component/server/etcd.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
```

Comment:
I think this was non-intentional due to a resolved merge conflict. I think we shouldn't use "path" here.